src/transforms.jl

```diff
@@ -40,15 +40,30 @@ clrinv(x) = Composition(𝓒(exp.(x)))
 
 Isometric log-ratio transformation of composition `c`.
 """
-function ilr(c::Composition)
-  # TODO
-end
+ilr(c::Composition{D,SYMS}) where {D,SYMS} =
+  SVector{D-1}(-helmert(D) * log.(c.parts .+ eps()))
 
 """
     ilrinv(x)
 
 Inverse ilr for coordinates `x`.
 """
-function ilrinv(x)
-  # TODO
+ilrinv(x::SVector{D,T}) where {D,T<:Real} =
+  Composition(SVector{D+1}(𝓒(exp.(-transpose(x) * helmert(D+1)))))
+
+"""
+  helmert(n)
+
+Create an Helmert matrix of order `n`.
+Based on `scipy.linalg.helmert` python code
+"""
+function helmert(n)
+  mat = zeros(n - 1, n)
+  for i in 1:n - 1
+      mat[i, 1:i] .= 1
+      mat[i, i + 1] = -i
+  end
+
+  d = [i * (i + 1) for i in 1.:n - 1.]
+  return mat ./ sqrt.(d)
```

Comment:
Can you please remove the explicit `return`?

Reply:
Sure!